tests/cephfs/ceph-83574182.py

```diff
@@ -0,0 +1,39 @@
+import logging
+import traceback
+
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+
+log = logging.getLogger(__name__)
+
+
+def run(ceph_cluster, **kw):
+
+    """
+    Test operation:
+    1. Create a subvolume name that does not exist
+    2. Try to remove the subvolume name that does not exist
+    3. The command should fail because the subvolume name does not exist.
+    4. Check if the command is failed
+    """
+    try:
+
```

Comment:
Can you remove extra line

Reply:
@AmarnatReddy I have deleted the extra line please review